pkg/controller.v1/common/job_test.go

```diff
@@ -219,6 +220,39 @@ func TestPastActiveDeadline(T *testing.T) {
 	}
 }
 
+func TestManagedBy(T *testing.T) {
+	cases := map[string]struct {
+		managedBy *string
```

Comment:
```suggestion
		managedBy *string
		wantResult bool
```
And provide true/false per test case depending if external controller.